src/routes/chart/get.ts

```diff
@@ -17,8 +17,11 @@ interface Response {
   series: {
     pinHashId: string;
     quantityHashId: string;
-    linkHashId: string;
-    measurementsSummary: {time: Date, value: number}[];
+    measurementsSummary: {
+      time: Date;
+      value: number;
+      linkHashId: string;
```

Comment:
a) What's the reason for including it here instead of one level higher up?
b) What's the reason for including it at all?

Reply:
c) if we include it, it should have `a1b2c3` as example instead of `po177`.